src/Application/GameConfig.cpp

```diff
@@ -19,3 +24,50 @@ MM_DEFINE_ENUM_SERIALIZATION_FUNCTIONS(PlatformWindowMode, CASE_INSENSITIVE, {
 
 GameConfig::GameConfig() = default;
 GameConfig::~GameConfig() = default;
+
+const std::vector<std::string> defaultCommands = {
```

Comment:
just do a `static constexpr std::initializer_list<const char *>`?

Reply:
Done, I had to change the constructor code a bit because initializer_list doesn't have a `operator[]` but it's still iterable.